src/test/java/com/hyunec/cosmicbaseballinit/acceptancetest/NormalBaseballLv1Test.java

```diff
@@ -1,36 +1,140 @@
 package com.hyunec.cosmicbaseballinit.acceptancetest;
 
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
 import org.junit.jupiter.api.DisplayName;
 import org.junit.jupiter.api.Test;
 
+
+import com.hyunec.cosmicbaseballinit.domain.baseball.model.Batting;
+import com.hyunec.cosmicbaseballinit.domain.baseball.model.BattingResult;
+import com.hyunec.cosmicbaseballinit.domain.baseball.model.PlateAppearances;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import com.hyunec.cosmicbaseballinit.domain.baseball.exception.ExceptionMessage;
+
+import java.util.Random;
+
+
 class NormalBaseballLv1Test {
+
+
     @DisplayName("strike, ball, hit 는 같은 확률 입니다.")
     @Test
     void t1() {
-        throw new RuntimeException("Not yet implemented");
+
+        int nDefalutNo = 100;
+        Random random = new Random();
+        int n1 = 0, n2 = 0, n3 = 0;
+        for (int i = 0; i < nDefalutNo; i++) {
+            Batting nResult = Batting.generate();
+            if (nResult == Batting.BALL) {
+                n1++;
+            }
+
+            if (nResult == Batting.STRIKE) {
+                n2++;
+            }
+
+            if (nResult == Batting.HIT) {
+                n3++;
+            }
+        }
+
+        System.out.println(
+                "[ BALL ]" + Math.floor((double) n1 / nDefalutNo * nDefalutNo)
+                        + " -- "
+                        + "[ STRIKE ]" + Math.floor((double) n2 / nDefalutNo * nDefalutNo)
+                        + " -- "
+                        + "[ HIT ]" + Math.floor((double) n3 / nDefalutNo * nDefalutNo)
+        );
+
+        boolean result = compareThreeValues((int) n1, (int) n2, (int) n3);
+        assertEquals(result, true);
```

Comment:
```suggestion
        assertTrue(result);
```
`nit` 저는 일관성을 위해 assertThat 의 활용을 추천합니다.